bootstrap-salt.sh

```diff
@@ -5843,15 +5843,15 @@ install_freebsd_git_deps() {
 
     if [ "${_POST_NEON_INSTALL}" -eq $BS_FALSE ]; then
 
-        SALT_DEPENDENCIES=$(/usr/local/sbin/pkg rquery %dn py37-salt)
+        SALT_DEPENDENCIES=$(/usr/local/sbin/pkg rquery %dn py38-salt)
         # shellcheck disable=SC2086
         /usr/local/sbin/pkg install -y ${SALT_DEPENDENCIES} python || return 1
 
-        /usr/local/sbin/pkg install -y py37-requests || return 1
-        /usr/local/sbin/pkg install -y py37-tornado4 || return 1
+        /usr/local/sbin/pkg install -y py38-requests || return 1
+        /usr/local/sbin/pkg install -y py38-tornado4 || return 1
```

Comment:
When searching for the packages, I found other options available here:

```console
# pkg search py38-tornado
py38-tornado-6.1               Python Web Server from FriendFeed
py38-tornado4-4.5.3            Python Web Server from FriendFeed
py38-tornado5-5.1.1            Python Web Server from FriendFeed
```

Can/should this be updated to use `py38-tornado5` instead?

Reply:
It's a bit more than that, we've to s/py37/py38 globally, I'd run it with tornado4 for the future